functions/src/index.ts

```diff
@@ -102,3 +102,90 @@ export const scheduledDeleteOfDocumentsSetToBeDeleted = region('us-central1')
         }
         return null
     })
+
+export const getOwnersDataByBoardIdAsOwner = https.onCall(
+    async (data, context) => {
+        if (!context.auth) {
+            throw new https.HttpsError(
+                'failed-precondition',
+                'The function must be called while authenticated.',
+            )
+        }
+
+        const boardRef = getFirestore().collection('settings').doc(data.boardId)
+        const boardSnapshot = await boardRef.get()
+        if (!boardSnapshot.exists) {
+            throw new https.HttpsError(
+                'invalid-argument',
+                'Requested board was not found.',
+            )
+        }
+        const boardOwners = boardSnapshot.data()?.owners
+        if (!boardOwners.includes(context.auth?.uid)) {
+            throw new https.HttpsError(
+                'permission-denied',
+                'You need to be an owner of this board to get data about other owners.',
+            )
+        }
+
+        const boardOwnersMapped = boardOwners.map((id: string) => ({ uid: id }))
+
+        try {
+            const usersResult = await getAuth().getUsers(boardOwnersMapped)
+            const ownersData = usersResult.users.map((user) => ({
+                uid: user.uid,
+                email: user.email,
+            }))
+            return ownersData
+        } catch {
+            throw new https.HttpsError(
+                'invalid-argument',
+                'Could not get requested data about owners.',
+            )
+        }
+    },
+)
+
+export const addInvitedUserToBoardOwners = https.onCall(
+    async (data, context) => {
+        if (!context.auth) {
+            throw new https.HttpsError(
+                'failed-precondition',
+                'The function must be called while authenticated.',
+            )
+        }
+
+        const boardRef = getFirestore().collection('settings').doc(data.boardId)
+        const boardSnapshot = await boardRef.get()
+        if (!boardSnapshot.exists) {
+            throw new https.HttpsError(
+                'invalid-argument',
+                'Requested board was not found.',
+            )
+        }
+        const boardOwners = boardSnapshot.data()?.owners
+
+        await getFirestore()
+            .collection('settings/' + data.boardId + '/invites')
```

Comment:
Trur du kan skrive dette sånn, for å unngå å sette sammen riktig streng:
```
.collection('settings')
.doc(data.boardId)
.collection('invites')
```

Reply:
Flotters! Det er bedre ja:)